ostd/src/mm/page_table/node/entry.rs

```diff
@@ -192,9 +192,20 @@ impl<'a, 'rcu, C: PageTableConfig> Entry<'a, 'rcu, C> {
         // Lock before writing the PTE, so no one else can operate on it.
         let mut pt_lock_guard = pt_ref.lock(guard);
 
+        let item = unsafe { C::item_from_raw(pa, level, prop) };
+        let parent_item = C::split_item(item);
+        core::mem::forget(parent_item);
```

Comment:
Why do we forget this? Doesn't it leak a frame? This is probably me not knowing something.